demos/python_demos/object_detection_retinaface_demo/postprocessor.py

```diff
@@ -0,0 +1,250 @@
+            type_scores_outputs.sort(key=lambda x: x.shape[1])
+
+        proposals_list = []
+        scores_list = []
+        landmarks_list = []
+        mask_scores_list = []
+        for idx, s in enumerate(self._features_stride_fpn):
+            anchor_num = self._num_anchors[s]
+            scores = self._get_scores(scores_outputs[idx], anchor_num)
+            bbox_deltas = bboxes_outputs[idx]
+            height, width = bbox_deltas.shape[1], bbox_deltas.shape[2]
+            anchors_fpn = self._anchors_fpn[s]
+            anchors = self.anchors_plane(height, width, int(s), anchors_fpn)
+            anchors = anchors.reshape((height * width * anchor_num, 4))
+            proposals = self._get_proposals(bbox_deltas, anchor_num, anchors)
+            x_mins, y_mins, x_maxs, y_maxs = proposals.T
+            keep = self.nms(x_mins, y_mins, x_maxs, y_maxs, scores, 0.5)
+            proposals_list.extend(proposals[keep])
+            scores_list.extend(scores[keep])
+            landmarks = self._get_landmarks(landmarks_outputs[idx], anchor_num, anchors)[keep, :]
+            landmarks_list.extend(landmarks)
+            if self._detect_masks:
+                mask_scores_list.extend(self._get_mask_scores(type_scores_outputs[idx], anchor_num)[keep])
+        scores = np.reshape(scores_list, -1)
+        mask_scores = np.reshape(mask_scores_list, -1)
+        x_mins, y_mins, x_maxs, y_maxs = np.array(proposals_list).T # pylint: disable=E0633
+        detections = [scores, x_mins / scale_x, y_mins / scale_y, x_maxs / scale_x, y_maxs / scale_y]
+
+        output = {}
+        output['face_detection'] = detections
+        if self._detect_masks:
+            output['mask_detection'] = [
+                scores, mask_scores, x_mins / scale_x,
+                y_mins / scale_y, x_maxs / scale_x, y_maxs / scale_y
+            ]
+
+        landmarks_x_coords = np.array(landmarks_list)[:, :, ::2].reshape(len(landmarks_list), -1) / scale_x
+        landmarks_y_coords = np.array(landmarks_list)[:, :, 1::2].reshape(len(landmarks_list), -1) / scale_y
+        output['landmarks_regression'] = [landmarks_x_coords, landmarks_y_coords]
+        return output
+
+    def _get_proposals(self, bbox_deltas, anchor_num, anchors):
+        bbox_deltas = bbox_deltas.transpose((1, 2, 0))
+        bbox_pred_len = bbox_deltas.shape[2] // anchor_num
+        bbox_deltas = bbox_deltas.reshape((-1, bbox_pred_len))
+        proposals = self.bbox_pred(anchors, bbox_deltas)
+        return proposals
+
+    @staticmethod
+    def _get_scores(scores, anchor_num):
+        scores = scores[anchor_num:, :, :]
+        scores = scores.transpose((1, 2, 0)).reshape(-1)
+        return scores
+
+    @staticmethod
+    def _get_mask_scores(type_scores, anchor_num):
+        mask_scores = type_scores[anchor_num * 2:, :, :]
+        mask_scores = mask_scores.transpose((1, 2, 0)).reshape(-1)
+        return mask_scores
```

Comment:
if I right remember only last class score used for mask
@Anna-Mironova  is it correct?
if yes, probably, it makes sense to store only mask probabilities